packages/integration-karma/test/component/dynamic-imports/index.spec.js

```diff
@@ -0,0 +1,99 @@
+import { createElement } from 'lwc';
+import Container from 'x/dynamic';
+import DynamicCtor from 'x/ctor';
+import AlterCtor from 'x/alter';
+import { registerForLoad, clearRegister } from 'test-utils';
+
+beforeEach(() => {
+    clearRegister();
+});
+
+function waitTwoMicroTasks(fn) {
+    return Promise.resolve().then(() => {
```

Comment:
@diervo this is failing in Safari and FF, works in Chrome... apparently there is a timing issue. I was trying to be very specific with the timing, waiting for the `await`, then waiting for the next tick of the engine, but apparently this is not good enough. Maybe we should just use macro-task instead, and forget about controlling the flow.